Microsoft.HealthVault/Thing/ThingBase.cs

```diff
@@ -835,7 +835,9 @@ public string Serialize()
             XDocument newDocument = SDKHelper.SafeLoadXml(GetItemXml());
 
             string dataName = "thing/data-xml";
-            fetchedDocument.Element(dataName).ReplaceWith(newDocument.Element(dataName));
```

Comment:
nvm, I thought that this method is being used for the update operation to platform.